### ISSUE TITLE
Discord "GPU process isn't usable"

### ISSUE THREAD

**Issue by wolfwood** [InBetweenNames/gentooLTO#805]
[emerge_info.txt](https://github.com/InBetweenNames/gentooLTO/files/7585369/emerge_info.txt)
not really sure if this is LTO related but I don't want to file an official bug until I can rule it out.  anyone else's discord stop launching? recently switched to gcc-11. Nvidia drivers and pipewire...

```wolfwood@BMO ~ $ discord
Discord 0.0.16
Starting app.
Starting updater.
[Modules] Modules initializing
[Modules] Distribution: remote
[Modules] Host updates: enabled
[Modules] Module updates: enabled
[Modules] Module install path: /home/wolfwood/.config/discord/0.0.16/modules
[Modules] Module installed file path: /home/wolfwood/.config/discord/0.0.16/modules/installed.json
[Modules] Module download path: /home/wolfwood/.config/discord/0.0.16/modules/pending
[26282:1122/165727.596912:FATAL:gpu_data_manager_impl_private.cc(415)] GPU process isn't usable. Goodbye.

Failed to get crash dump id.
Report Id: a002962a-6fdc-43

Failed to get crash dump id.
Report Id: 123741b7-86a3-4f
[1]    26282 illegal hardware instruction  discord
```

**Comment by wolfwood** [comment]
looks  like its an unspecified electron/nvidia issue, affecting other apps as well: https://www.reddit.com/r/openSUSE/comments/pvnhjb/electron_apps_fail_to_start_since_nvidia_47074/

TLDR: `discord  --disable-gpu-sandbox`
